HelpMyStreetFE/HelpMyStreetFE/src/js/profile/requests.js

```diff
@@ -85,11 +85,22 @@ export function initialiseRequests(isVerified) {
         buttonUnload($(this));
     });
 
+    $('.job-list').on('click', '.email-details', async function (evt) {
+        const job = $(this).closest('.job');
+        let jobId = job.attr("id");
+        let response = await hmsFetch(`/account/emailJobDetails?id=${jobId}`);
```

Comment:
Do we want to configure hmsFetch _not_ to retry?  We don't want people getting 5 copies of the same email from us, if our service is slow waking up.